CHANGES.rst

```diff
@@ -40,6 +40,8 @@ Changelog
 - Grouped task entries can now be sorted by start date, name, duration or
   according to tasks.txt order (GH: #228).
 
+- Add the ability to change the last entry using Primary+BackSpace (GH: #247).
```

Comment:
I find the GTK terminology of Primary+... confusing (and I think they dropped it for GTK 4?) so I would suggest

```suggestion
- Add the ability to change the last entry using Ctrl+Backspace (GH: #247).
```

instead

Reply:
On MacOS it is actually Command-Backspace, so I wrote `primary`. But I am also fine with `Ctrl`.

Reply:
Hm, MacOS was probably why Gtk introduced those `Primary+` etc. modifiers.  That's fine then.